docs/whats-new.mdx

```diff
@@ -19,6 +20,8 @@ Target Release Date: April, 2024
 
 **@loaders.gl/parquet**
 
+- Support compression modes: 'zstd', 'snappy', 'gzip', 'brotli', 'no_compression'.
```

Comment:
Some of these were already supported? Certainly no_compression, gzip and snappy?

Reply:
I don't remember right now. Daria made fixes for few types of compressions.